src/sections/gitops/index.js

```diff
@@ -38,6 +40,38 @@ const GitOpsPage = () => {
         </Row>
       </div>
 
+      <div className="content-container content-container-more-width">
+        <Row className="content-row">
+          <Col lg={6} md={6} className="desc-text-container">
+            <h1 className="heading-1">What is GitOps?</h1>
+            <SquarePoint>
+              GitOps uses Git for defining and tracking infrastructure and app
+              changes.
+            </SquarePoint>
+            <SquarePoint>
+              It automates syncing actual state with Git's desired state.
```

Comment:
It’s an evolution of Infrastructure as Code (IaC) that leverages Git as the single source of truth, and control mechanism for creating, updating, and deleting system architecture.

These lines can be added.

Reply:
Ohk i will add them